spec/features/api_access_spec.rb

```diff
@@ -2,6 +2,13 @@
 require 'json'
 
 RSpec.describe "api access", :type => :request do
+  def sign_in_as_admin
+    @admin ||= FactoryGirl.create(:admin)
+    post_via_redirect(admin_session_path,
+                      'admin[email]' => @admin.email,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.